pyiode/objects/table.pyx

```diff
@@ -249,6 +255,7 @@ cdef class TableCell:
         if self.c_cell is not NULL:
             self.c_cell.set_underline(<bint>value)
 
+    # GDM> I feel like this should be a .get_variables() method (or .variables property) on a LEC (or Lec, or whatever the name) class
```

Comment:
I will just make the method *get_variables_from_lec()* a property *variables* for the moment.